src/storage/store/rel_table.cpp

```diff
@@ -192,17 +190,17 @@ vector<unique_ptr<ListsUpdateIterator>> DirectedRelTableData::getListsUpdateIter
     vector<unique_ptr<ListsUpdateIterator>> listsUpdateIterators;
     listsUpdateIterators.push_back(
         ListsUpdateIteratorFactory::getListsUpdateIterator(adjLists.at(boundNodeTableID).get()));
-    for (auto& propList : propertyLists.at(boundNodeTableID)) {
-        listsUpdateIterators.push_back(
-            ListsUpdateIteratorFactory::getListsUpdateIterator(propList.get()));
+    for (auto i = 0u; i < propertyLists.at(boundNodeTableID).size(); i++) {
```

Comment:
For consistency, better do `auto& [, propertyList] : propertyLists.at(boundNodeTableID)`

Reply:
This function has been changed in my another PR, i just want to make it work in this pr